docs/blocks-storage/compactor.md

```diff
@@ -298,4 +298,8 @@ compactor:
   # When enabled, index verification will ignore out of order label names.
   # CLI flag: -compactor.accept-malformed-index
   [accept_malformed_index: <boolean> | default = false]
+
+  # When enabled, will fetch the list of block ids using bucket index.
+  # CLI flag: -compactor.bucket-index-block-ids-fetcher-enabled
+  [bucket_index_block_ids_fetcher_enabled: <boolean> | default = false]
```

Comment:
Do we want to give this level of config to the user? Is it common on the StoreGateway side config? I image if this works fine we should just use it as it improves quite a lot the compaction time. @yeya24 maybe you can give some guidance on it

I see there is a TODO in the code.
I think there is a pattern to use `experimental` flags mostly adding it to the description of the config, but i am not sure this need to be experimental.

Reply:
removed the feature flag. Now the blockindex block ids fetcher will be use as default if the bucketindex is enabled.